lighthouse-core/report/v2/report-renderer.js

```diff
@@ -199,8 +279,10 @@ window.ReportRenderer = class ReportRenderer {
    * @return {!Element}
    */
   _renderCategory(category) {
-    const element = this._createElement('div', 'lighthouse-category');
-    element.appendChild(this._renderScore(category.score, category.name, category.description));
+    const element = this._dom.createElement('div', 'lighthouse-category');
+    const score = {value: Math.round(category.score), scoringMode: 'numeric'};
```

Comment:
just kidding, you do need `scoringMode` for the class name, but still seems like they should be separate parameters :)